README.md

```diff
@@ -250,6 +250,30 @@ probably do some careful consideration before you do so.
 
 This function is not exportable, even on request.
 
+## Extra library paths
+
+### macOS
+
+On macOS platforms with [Homebrew](http://brew.sh) and/or [MacPorts](https://www.macports.org)
+package managers installed, default manager's lib paths will be automatically appended to
+`$system_path`. In case of having both managers installed, Homebrew will appear before.
+
+This behaviour can be overridden using the environment variable `FFI_CHECKLIB_PACKAGE`.
+
+Allowed values are:
+
+- `none`: Won't use either Homebrew path nor MacPorts
+- `homebrew`: Will append the `$(brew --prefix)/lib` to the system paths
+- `macports`: Will append `port` default lib path
+
+A comma separated list is also valid:
+
+```sh
+export FFI_CHECKLIB_PACKAGE=macports,homebrew
+```
+
+Order matters, so in this example, MacPorts' lib path appears before Homebrew's path.
+
```

Comment:
Looks like you made changes directly to the README.md, but this is generated by the POD in `lib/FFI/CheckLib.pm` (this is understandably not necessarily obvious).  Can you please put this in the .pm file in POD format instead?

Reply:
Of course, I didn't realize about that!

Reply:
Done!